qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -185,6 +198,53 @@ def num_qubits(self) -> int:
         """
         return len(self.qubits)
 
+    def draw_map(
+        self,
+        backend: BackendV2,
+        coordinates: Optional[list[list[int]]] = None,
+        *,
+        colorscale: str = "Bluered",
+        color_no_data: str = "lightgray",
+        edges_n_segs: int = 16,
```

Comment:
qiskit tends to use the full `num` for numbers.

```suggestion
        edges_num_segs: int = 16,
```

or how about the more verbose

```suggestion
        num_edge_segments: int = 16
```